### DISCUSSION TITLE
Cannot find module poppinss

### DISCUSSION THREAD

**Discussion by crrmacarse** [adonisjs/core#1434] [Help] [answered]
My auth started failing on production after updating packages

```
    "@adonisjs/ace": "^6.9.3",
    "@adonisjs/auth": "^4.3.1",
    "@adonisjs/core": "^5.0.0-preview-rc-1.11",
    "@adonisjs/fold": "^6.3.5",
    "@adonisjs/lucid": "^8.3.1",
    "@adonisjs/session": "^3.0.8",
    "@adonisjs/shield": "^3.0.6",
    "@adonisjs/validator": "^7.4.0",
    "@adonisjs/view": "^2.0.3",
```

```Cannot find module '@poppinss/utils'↵Require stack:↵- /app/build/app/Exceptions/ErrorException.js↵- /app/node_modules/@adonisjs/fold/build/src/Ioc/index.js↵- /app/node_modules/@adonisjs/fold/build/index.js↵- /app/node_modules/@adonisjs/core/build/src/Ignitor/Bootstrapper/index.js↵- /app/node_modules/@adonisjs/core/build/src/Ignitor/Ace/AppCommands.js↵- /app/node_modules/@adonisjs/core/build/src/Ignitor/Ace/index.js↵- /app/node_modules/@adonisjs/core/build/src/Ignitor/index.js↵- /app/build/server.js"```

locally run seems fine

**Comment by thetutlage** [comment, marked as answer]
The error is coming from the `app/Exceptions/ErrorException` file. Can you please share the contents of this file?

**Reply by crrmacarse** [reply]
It was based from the docs:
https://preview.adonisjs.com/guides/http/exception-handling

```
import { Exception } from '@poppinss/utils'
import { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'

/**
 * Error Exception
 *
 * @USAGE
 * throw new ErrorException('message', 500, 'E_RUNTIME_EXCEPTION')
 */
export default class ErrorException extends Exception {
  constructor (message: string, code: number = 400) {
    super(message, code)
  }

  public async handle (error: this, { response }: HttpContextContract) {
    response.status(error.status).send({
      status: 'error',
      message: error.message,
    })
  }
}

```
I probably need to install the package? There seems to be no note about installing though

**Reply by thetutlage** [reply]
Yes, you have to. Docs can be improved here. Can you please share a PR for the same in [adonisjs.com repo](https://github.com/adonisjs/adonisjs.com)

**Reply by crrmacarse** [reply]
Will do. Thanks and more power for the adonis team 💪
